operator/config/crd/bases/resources.teleport.dev_roles.yaml

```diff
@@ -0,0 +1,978 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
I think we can remove this file
They are copied into the subchart

The same goes to the Teleport User CRD

Reply:
Related to the discussion: https://github.com/gravitational/teleport/pull/13331#discussion_r917608473